apollo-router/tests/integration/query_planner.rs

```diff
@@ -0,0 +1,443 @@
+use std::path::PathBuf;
+
+use crate::integration::common::graph_os_enabled;
+use crate::integration::IntegrationTest;
+
+const PROMETHEUS_METRICS_CONFIG: &str = include_str!("telemetry/fixtures/prometheus.router.yaml");
```

Comment:
added prometheus to the config so we can also check that the metrics are emitted for the reload integration tests.